docs/pages/branding/about.tsx

```diff
@@ -2,10 +2,12 @@ import * as React from 'react';
 import BrandingRoot from 'docs/src/modules/branding/BrandingRoot';
 import BrandingBeginToday from 'docs/src/modules/branding/BrandingBeginToday';
 import BrandingDiscoverMore from 'docs/src/modules/branding/BrandingDiscoverMore';
+import BrandingSupportTheProject from 'docs/src/modules/branding/BrandingSupportTheProject';
 
 export default function Page() {
   return (
     <BrandingRoot>
+      <BrandingSupportTheProject />
```

Comment:
A note, I have created components for the other blocks because they needed to be shared with other pages. It seems that this specific block is never shared, at least not in its default format. In the pricing page, we have the following block:

<img width="542" alt="Capture d’écran 2021-01-12 à 19 35 44" src="https://user-images.githubusercontent.com/3165635/104357381-65747980-550d-11eb-89e3-c60871824215.png">

With the exact same layout. Maybe we could create a component for it, and only fill the gaps for the about page.

Also note, that we have used this block to test the skills of the freelancer that would handle all the pages, so we might have some duplication, and maybe we don't need to care or try to be DRY.

Reply:
I see, we can probably at least reuse the cards... We can probably extract later on the `SupportCard` to something more general when working on the next page...

As this block is not shared with other pages, I can at least rename it to `BrandingAboutSupportTheProject` maybe?

Reply:
Or did you mean we should just inline this in the `about.tsx` content?

Reply:
> Or did you mean we should just inline this in the about.tsx content?

As you prefer. I'm personally the type of developer that would enjoy a 3k LOCs file. I was more interested in raising the existence of the potential duplication.

Reply:
If nothing else, it'll be interesting the compare with the freelancer's approach.

Reply:
I've extracted the card into it's own component `BrandingCard` and moved the rest of the content in the `about.tsx` page. It would be easier for finding the content and navigating in the future this way

Reply:
Sounds great